client/client.js

```diff
@@ -82,6 +82,9 @@ Client.prototype = {
     this.post('/institutions/' + institutionId + '/logins', answers, extraHeaders)
       .then(function(response){
         deferred.resolve(response);
+      },
+      function(reason){
+        deferred.reject('Mult-Factor Authentication failed.  Message from Intuit: ' + reason);
```

Comment:
Multi is spelled wrong.

Since this endpoint is shared, do we know if this is always the reason?